src/store/modules/Config.ts

```diff
@@ -1,36 +1,38 @@
 import { randomHex } from '@/modules/encryption/crypto';
-// import defaultConfig from '@/originalBudgetTrackingApp/configManager/defaultConfig';
+import { Config } from '@/originalBudgetTrackingApp/configManager/configManager';
+import defaultConfig from '@/originalBudgetTrackingApp/configManager/defaultConfig';
+import { ActionTree, GetterTree, MutationTree } from 'vuex';
 
 export const ADD_IMPORTER_ACTION = 'ADD_IMPORTER_ACTION';
 export const ADD_EXPORTER_ACTION = 'ADD_EXPORTER_ACTION';
 export const REMOVE_IMPORTER_ACTION = 'REMOVE_IMPORTER_ACTION';
 export const GET_IMPORTERS_GETTER = 'GET_IMPORTERS_GETTER';
 export const GET_EXPORTER_GETTER = 'GET_EXPORTER_GETTER';
 
-const state = {};
+const state: Config = defaultConfig;
 
-const mutations = {
-  addImporter: (state, importer) => state.scraping.accountsToScrape.push(importer),
-  removeImporter: (state, importerId) => {
+const mutations = <MutationTree<Config>>{
+  addImporter: (state: Config, importer) => state.scraping.accountsToScrape.push(importer),
+  removeImporter: (state: Config, importerId) => {
```

Comment:
```suggestion
  removeImporter: (state: Config, importerId: string) => {
```

Reply:
Is this fix because of any error, or just for use types everywhere we can?

Reply:
Just to add the type since we know what it is

Reply:
OK, I will do it from now on